docs/content/how-to/aws/create-aws-hosted-cluster-secret-credentials.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Create an AWS Hosted Cluster using a credential secret
+---
+
+# Create an AWS Hosted Cluster using a credential secret
+
+## Creating the credential secret
+
+If using the Multi-cluster Engine, the console provides a credential page that allows you to create the AWS credential secret.
+
+To manually create the secret, apply the following secret yaml:
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: my-aws-credentials
+  namespace: clusters
+  labels:
+    cluster.open-cluster-management.io/credentials: ""
+    cluster.open-cluster-management.io/type: aws
+stringData:
```

Comment:
what is the historical context for keeping the base domain and the ssh info in this same secret?

Reply:
ACM/MCE hive provisioning, this has been the existing flow for a while, both in the console and the automated flows. We have asks to support multiple secrets, but this is not from a majority of our user base.

We are looking at revisiting this at some point in the future:
1. In the console flows should we allow mulitple credentials to be referenced and each reference only pulls the relevant keys.
2. Generate NEW credentials in ACM using existing credentials as a base (clone and edit)
3. Double check to see if there have been more then the one or two RFE's to justify the first two.
4. In our anecdotal data, for automated flows this often simplifies the number of secrets to manage and setup once.